.Rbuildignore

```diff
@@ -1,5 +1,5 @@
 ^run-*\.R$
-^test-*\.R$
+^./test-.*\.R$
```

Comment:
I wasn't able to figure out the point of this change.  Doesn't the `./` anchor this on a single character followed by a slash?  Intentional?